package.json

```
@@ -15,7 +15,8 @@
     "storybook:build": "build-storybook -s ./docs/assets -o ./docs/storybook",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "chromatic": "chromatic --project-token"
```

What would the flag `--project-token` do without a value?

```suggestion
    "chromatic": "chromatic"
```

---

I guess I forgot to push the changes. I made some tests and ran the the script from command line so I used the token saved in env variables and I missed to delete the flag as well